src/inference/src/ie_core.cpp

```diff
@@ -1113,7 +1143,18 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
 
     std::map<std::string, std::string> GetSupportedConfig(const std::string& deviceName,
                                                           const std::map<std::string, std::string>& configs) override {
-        std::vector<std::string> supportedConfigKeys = GetMetric(deviceName, METRIC_KEY(SUPPORTED_CONFIG_KEYS));
+        std::vector<std::string> supportedConfigKeys;
+        try {
+            auto supported_properties = ICore::get_property(deviceName, ov::supported_properties);
```

Comment:
`ov::supported_properties` does not provide old properties for most of the plugins. So, we need to combine mutable properties from `ov::supported_properties` plus `METRIC_KEY(SUPPORTED_CONFIG_KEYS)`